php/hireinsocial/resources/templates/pl_PL/ui/specialization/_name.html.twig

```diff
@@ -0,0 +1,4 @@
+{% spaceless %}
+{% if specialization == 'php' %}PHP{% endif %}
+{% if specialization == 'js' %}Java Script{% endif %}
+{% endspaceless %}
```

Comment:
I don't want to keep SEO content and other "descriptions" in database, I also don't want to create any limitations for the user interface designer. 
Because of that I'm going to put everything directly into the twig. It's not most flexible approach, adding new specializations will require some coding but at this point it's smaller price than building CRUD mechanism for adding SEO content that might never be used.